CHANGES.md

```diff
@@ -14544,8 +14506,8 @@ ndif
 
    *Steve Henson*
 
- * crypto/err.c locking bugfix: Use write locks (CRYPTO_w_[un]lock),
-   not read locks (CRYPTO_r_[un]lock).
+ * crypto/err.c locking bugfix: Use write locks (`CRYPTO_w_[un]lock)`,
```

Comment:
Last tick is misplaced: `` (`CRYPTO_w_[un]lock)` ``  ->  `` (`CRYPTO_w_[un]lock`) ``

Reply:
> Last tick is misplaced: (`CRYPTO_w_[un]lock)` -> (`CRYPTO_w_[un]lock`)
fixed.